docs/content/Getting-Started/user-quickstart-kind.md

```diff
@@ -0,0 +1,276 @@
+   
+#### 1. Deploy the <span class="Space-Bd-BT">KUBESTELLAR</span> Core component  
+{%
+  include-markdown "../common-subs/deploy-your-kubestellar-core-component-kind.md"
+  start="<!--deploy-your-kubestellar-core-component-kind-start-->"
+  end="<!--deploy-your-kubestellar-core-component-kind-end-->"
+%}
+
+#### 2. Install <span class="Space-Bd-BT">KUBESTELLAR</span>'s user commands and kubectl plugins
+!!! tip ""
+    === "install"
+         {%
+           include-markdown "../common-subs/brew-install.md"
+           start="<!--brew-install-start-->"
+           end="<!--brew-install-end-->"
+         %}
+    === "remove"
+         {%
+           include-markdown "../common-subs/brew-remove.md"
+           start="<!--brew-remove-start-->"
+           end="<!--brew-remove-end-->"
+         %}
+
+#### 3. View your <span class="Space-Bd-BT">KUBESTELLAR</span> Core Space environment
+!!! tip ""
+    === "show all available <span class="Space-Bd-BT">KUBESTELLAR</span> Core Spaces"
+         {%
+           include-markdown "../common-subs/kubestellar-show-available-spaces.md"
+           start="<!--kubestellar-show-available-spaces-start-->"
+           end="<!--kubestellar-show-available-spaces-end-->"
+         %}
+    
+    === "uh oh, error?"
+         Did you received the following error:
+         ```Error: Get "https://some_hostname.some_domain_name:{{config.ks_kind_port_num}}/clusters/root/apis/tenancy.kcp.io/v1alpha1/workspaces": dial tcp: lookup some_hostname.some_domain_name on x.x.x.x: no such host``
+
+         A common error occurs if you set your port number to a pre-occupied port number and/or you set your EXTERNAL_HOSTNAME to something other than "localhost" so that you can reach your <span class="Space-Bd-BT">KUBESTELLAR</span> Core from another host, check the following:
+         
+         Check if the port specified in the **ks-core** Kind cluster configuration and the EXTERNAL_PORT helm value are occupied by another application:
+
+         &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1. is the port specified in this example occupied by another process?  If so, delete the **ks-core** Kind cluster and create it again using an available port for your 'hostPort' value
+
+         &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2. if you change the port for your **ks-core** 'hostPort', remember to also use that port as the helm 'EXTERNAL_PORT' value
+
+         Check that your EXTERNAL_HOSTNAME helm value is reachable via DNS:
+
+         &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1. use 'nslookup <value of EXTERNAL_HOSTNAME>' to make sure there is a valid IP address associated with the hostname you have specified
+
+         &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2. make sure your EXTERNAL_HOSTNAME and associated ip address are listed in your /etc/hosts file.
+
+         &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3. make sure the IP address is associated with the system where you have deployed the **ks-core** Kind cluster
```

Comment:
We should also say that the IP address is one that can be used to open TCP connections from the syncers and the shell(s) where the user will be issuing commands. While it is true that 127.0.0.1 is an IP address that is "associated with the system where you have deployed the **ks-core** Kind cluster", it is not an IP address that EXTERNAL_HOSTNAME should resolve to.